### PR TITLE
Feature/cdr 18

### PR DESCRIPTION
**Pull request by chevalleyc**
## Changes
Make sure that table `entry` is used in the FROM clause whenever the template is not resolved (that is resulting in "null" field).

## Related issue

fixes: https://jira.vitagroup.ag/browse/CDR-18

## Additional information and checks

- [ ] Pull request linked in changelog

### CODE REVIEW

File: service/src/main/java/org/ehrbase/aql/sql/QueryProcessor.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import org.ehrbase.dao.access.interfaces.I_DomainAccess;
 import org.ehrbase.service.IntrospectService;
 import org.jooq.*;
+import org.jooq.Record;
```
Comment (subigre) [inline service/src/main/java/org/ehrbase/aql/sql/QueryProcessor.java:38]: Minor but we should either go with wildcard imports or either remove them but I think that we have to keep consistency in the code.

From personal point of view, I prefer the second option. In addition, it is compliant with Google Java style guide.

**Review by subigre — COMMENTED** [review]
The PR is not mentioned in `CHANGELOG.md `

File: service/src/main/java/org/ehrbase/aql/sql/QueryProcessor.java (hunk 1 above)
Comment (chevalleyc) [reply service/src/main/java/org/ehrbase/aql/sql/QueryProcessor.java:38]: I did this change as it was causing a compile error with the IDE (but not with Maven...). Basically, it was conflicting with a java.lang.Record. Not quite sure what is the issue TBH

Comment (subigre) [reply service/src/main/java/org/ehrbase/aql/sql/QueryProcessor.java:38]: You are right (One more reasons to avoid wildcard imports 😛 )

**Review by subigre — APPROVED** [review]

**Pull request merged**